src/main/java/org/folio/qm/domain/entity/JobProfile.java

```diff
@@ -41,15 +39,18 @@ public class JobProfile {
   @NotNull
   @Column(name = "profile_action", nullable = false)
   @Enumerated(EnumType.STRING)
-  @Type(PostgreSQLEnumType.class)
   private JobProfileAction profileAction;
 
   @NotNull
   @Column(name = "record_type", nullable = false)
   @Enumerated(EnumType.STRING)
-  @Type(PostgreSQLEnumType.class)
   private RecordType recordType;
 
+  @Override
+  public int hashCode() {
+    return getClass().hashCode();
```

Comment:
Why hashcode from class?

Reply:
https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/